internal/framework5provider/dynamic_computed_type_change_resource.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package framework
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var _ resource.Resource = DynamicComputedTypeChangeResource{}
+
+func NewDynamicComputedTypeChangeResource() resource.Resource {
+	return &DynamicComputedTypeChangeResource{}
+}
+
+// DynamicComputedTypeChangeResource is for testing the ability of a computed dynamic attribute type to change on apply (update) when unknown
+// Ref: https://github.com/hashicorp/terraform-plugin-framework/issues/969
+type DynamicComputedTypeChangeResource struct{}
+
+func (r DynamicComputedTypeChangeResource) Metadata(_ context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_dynamic_computed_type_change"
+}
+
+func (r DynamicComputedTypeChangeResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"required_dynamic": schema.DynamicAttribute{
+				Required: true,
+			},
+			// This computed dynamic attribute changes type during update
+			"computed_dynamic_type_changes": schema.DynamicAttribute{
+				Computed: true,
+			},
+			// id attribute is required for acceptance testing.
```

Comment:
Nit: Not since [terraform-plugin-testing v1.5.0](https://github.com/hashicorp/terraform-plugin-testing/releases/tag/v1.5.0) 😉

Reply:
It's crazy how a code comment can make me believe this still existed 😆 , time for the weekend!

⚡ -> 45bbdbf